Dockerfile

```diff
@@ -19,6 +19,8 @@ RUN ansible-playbook -v -c local -i localhost, ${WORKDIR}/files/install-requirem
 
 COPY ./ ${WORKDIR}/
 RUN pip3 install ${WORKDIR}/ \
+    && cd ${WORKDIR} && git rev-parse HEAD >/.packit.git.commit.hash \
+    && git show --quiet --format=%B HEAD >/.packit.git.commit.message \
```

Comment:
Is that better than the env-var? (Just curious.)

Reply:
honestly, how would you implement this with an env var? I'd say this is better since the data is stored inside the image.

Reply:
Just wondering if docker hub supports some build tweaking.

Reply:
github actions + ansible-bender :P

Reply:
Fell free to set it up..;)